util/template/expression_template_test.go

```diff
@@ -0,0 +1,17 @@
+package template
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
We need this to be formatted in a specific order:

```
import (

native libraries

external libraries

internal libraries
)
```

In this case it should be 
```go
import (
    "testing"

    "github.com/stretchr/testify/assert"
)
```